robinhood.spec.in

```diff
@@ -159,7 +157,7 @@ Annex tools for robinhood.
 %setup -q -n @PACKAGE@-%{version}
 
 %build
-./configure @ac_configure_args@ %{?configure_flags:configure_flags} \
```

Comment:
It is OK if it's build on a machine without lustre installed.
But if you do the same on a lustre host, the ./configure step will detect it and build robinhood with it.
It is not only about Lustre, the problem is still the same for other possible ./configure arguments that are not passed to the rpm build step.

I think the proposed change is nice, but to preserve the propagation of configure options to "make rpm" the Makefile.am should call rpmbuild with a "define" of configure_flags.
Then, the build is correctly done with ./configure  %{?configure_flags}.

BTW, the original syntax in the spec file was bad. I think It is just %{?configure_flags}